metfrag.sh

```diff
@@ -41,7 +41,8 @@ for filename in tmpmetf/*.*; do
 counter=$((counter+1))
 FIleOutName=$(basename ${filename%.*}.csv)
 
-java -Xmx2048m -Xms1024m -jar /usr/local/bin/MetFragCLI.jar "`cat $filename` ResultsFile=resultsmet/${FIleOutName} NumberThreads=1 $PARAM " LocalDatabasePath=${DB}&
+
+/usr/local/bin/metfrag -Xmx2048m -Xms1024m "`cat $filename` ResultsFile=resultsmet/${FIleOutName} NumberThreads=1 $PARAM " LocalDatabasePath=${DB}&
```

Comment:
`/usr/local/bin/metfrag` is a warpper defined here:
https://github.com/bioconda/bioconda-recipes/blob/master/recipes/metfrag/metfrag.sh